back/boxtribute_server/business_logic/user/fields.py

```diff
@@ -8,7 +8,17 @@
 
 
 @user.field("bases")
-def resolve_bases(*_):
+def resolve_bases(user_obj, _):
+    if user_obj.id != g.user.id:
+        # If the queried user is different from the current user, we don't have a way
+        # yet to fetch secure information about that user's bases; it would require
```

Comment:
So we will just throw an error or hit logins or...?

Reply:
we return `null` for the list of a user's bases

Reply:
So basically it needs to be handled by the front end right? @HaGuesto or @fhenrich33 can you confirm it's been handled?

Reply:
yes but it's a field that the FE does not use (yet). Only user ID and name are used